packages/react-composites/src/composites/CallComposite/hooks/useAdaptedSelector.ts

```diff
@@ -77,7 +71,7 @@ export const useSelectorWithAdaptation = <
     return () => {
       adapter.offStateChange(onStateChange);
     };
-  }, [adaptState, adapter, selector, selectorProps, callConfigProps]);
+  }, [adaptState, adapter, selector, selectorProps, callId]);
```

Comment:
shouldn't need callId here

Reply:
I thought we needed it there so that we trigger the hook appropriately?

Reply:
Question back to you - what would it be used for in this use effect?

Reply:
(the use effect doesn't trigger the selector, the useeffect here just sets up the onstatechange subscription)

Reply:
right thats true. yeah we can remove it because the useEffect pass and add the event subscription so technically nothing in the array causes the hook to fire, the onstate change does.

Reply:
right that makes sense, the hook needs to be concerned about the changes to the adapter and selector to make sure the event is subscribed to the right adapter. so doesn't need the callId since that would be fetched anytime there is a re-render triggering another call to useSelector.